src/app/shared/issue-tables/issue-tables.component.ts

```diff
@@ -99,6 +101,17 @@ export class IssueTablesComponent implements OnInit {
     event.stopPropagation();
   }
 
+  isResponseEditable() {
+     switch (this.phaseService.currentPhase) {
+       case Phase.phaseBugReporting:
+       case Phase.phaseModeration:
+         case Phase.phaseTeamResponse:
```

Comment:
indentation error here.

Reply:
Actually can the function be simplified to something like this? 

```javascript
this.permissions.isTeamResponseEditable() || this.permissions.isTesterResponseEditable()
```

Reply:
Thanks, fixed it 👍

Reply:
>Actually can the function be simplified to something like this? 
>
>```javascript
>return this.permissions.isTeamResponseEditable() || this.permissions.isTesterResponseEditable()
>```

Yes I thought of that initially, but moving forward wasn't exactly sure if we are adding any other requirements to check if a particular issue is editable. But sure I'll just change it to the suggestion 👍

Reply:
Thanks @ptvrajsk !